apps/fyle/migrations/0041_expense_expenses_workspa_ad984e_idx.py

```
@@ -0,0 +1,17 @@
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('fyle', '0040_alter_expense_imported_from'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='expense',
+            index=models.Index(fields=['workspace_id', 'report_id'], name='expenses_workspa_ad984e_idx'),
+        ),
+    ]
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)